### ISSUE TITLE
Accessibility: Ordered List & Unordered List dropdown buttons require a label

### ISSUE THREAD

**Issue by a11y1** [froala/wysiwyg-editor#4439]
##### Expected behavior.
The Ordered List & Unordered List dropdown buttons require a label. 
The screenreader should announce the specific purpose of the button, such as "Unordered List Options" and "Ordered List Options".

##### Actual behavior.
The Ordered List & Unordered List dropdown buttons do not have a label or discernible text. 
Currently, the screenreader announces "menu dropdown" only. There is not enough information to ascertain what the button does. 

##### OS.
Mac

##### Browser.
Chrome

##### Recording.
(A recording showing how to reproduce the problem)

**Comment by a11y1** [comment]
![Screen Shot 2022-04-21 at 2 48 16 PM](https://user-images.githubusercontent.com/102971546/164534463-210cc83d-cfc6-4c12-a59e-ce861ca9af4c.png)
